RUFAS/biophysical/animal/digestive_system/enteric_methane_calculator.py

```diff
@@ -243,53 +236,47 @@ def _calculate_lactating_cow_enteric_methane(
-                methane_emission = (
-                    -126
-                    + 11.3 * dry_matter_intake
-                    + 2.30 * neutral_detergent_fiber_concentration
-                    + 28.8 * milk_fat
-                    + 0.148 * body_weight
-                )
-                methane_results[model] = methane_emission
-            elif model == "Mills" and selected:
-                mitscherlich_parameter_a = animal_constants.MITS_PARAMETER_A
-                mitscherlich_parameter_b = animal_constants.MITS_PARAMETER_B
-                mitscherlich_parameter_c = -0.0011 * starch_concentration / acid_detergent_fiber_concentration + 0.0045
-                methane_emission_MJ = mitscherlich_parameter_a - (
-                    mitscherlich_parameter_a + mitscherlich_parameter_b
-                ) * exp(-mitscherlich_parameter_c * metabolizable_energy_intake * GeneralConstants.KCAL_TO_MJ)
-                methane_emission = methane_emission_MJ / GeneralConstants.MJ_CH4_TO_G_CH4
-                methane_results[model] = methane_emission
-            elif model == "IPCC" and selected:
-                soluble_residue = (
-                    GeneralConstants.FRACTION_TO_PERCENTAGE
-                    - ash_concentration
-                    - neutral_detergent_fiber_concentration
-                    - crude_protein_concentration
-                    - ethyl_ester_concentration
-                )
-                gross_energy_concentration = (
-                    0.263 * crude_protein_concentration
-                    + 0.522 * ethyl_ester_concentration
-                    + 0.198 * neutral_detergent_fiber_concentration
-                    + 0.160 * soluble_residue
-                )
-                methane_emission = (
-                    0.065 * gross_energy_concentration * dry_matter_intake / GeneralConstants.MJ_CH4_TO_G_CH4
-                )
-                methane_results[model] = methane_emission
-
-        return methane_results
+        if methane_model == "Mutian":
+            methane_emission = (
+                -126
+                + 11.3 * dry_matter_intake
+                + 2.30 * neutral_detergent_fiber_concentration
+                + 28.8 * milk_fat
+                + 0.148 * body_weight
+            )
+            return methane_emission
+        elif methane_model == "Mills":
+            mitscherlich_parameter_a = animal_constants.MITS_PARAMETER_A
+            mitscherlich_parameter_b = animal_constants.MITS_PARAMETER_B
+            mitscherlich_parameter_c = -0.0011 * starch_concentration / acid_detergent_fiber_concentration + 0.0045
+            methane_emission_MJ = mitscherlich_parameter_a - (
+                mitscherlich_parameter_a + mitscherlich_parameter_b
+            ) * exp(-mitscherlich_parameter_c * metabolizable_energy_intake * GeneralConstants.KCAL_TO_MJ)
+            methane_emission = methane_emission_MJ / GeneralConstants.MJ_CH4_TO_G_CH4
+            return methane_emission
```

Comment:
It looks like we have this duplicated in `_calculate_dry_cow_enteric_methane()` and `_calculate_lactating_cow_enteric_methane()` - maybe extract each methane model calculation into its own function for clarity and ease of re-use and also testing?